teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission: parent_permission)
```

Comment:
- [ ] it would be better to remove the id for more details check the comment above

Reply:
Thank you for the advice! I'm learning Ruby and didn't know most of the things I needed to correct.